modules/dnn/test/test_halide_layers.cpp

```diff
@@ -167,6 +167,13 @@ TEST_P(Deconvolution, Accuracy)
         throw SkipTestException("Test is disabled for OpenVINO 2018R4");
 #endif
 
+#if defined(INF_ENGINE_RELEASE) && INF_ENGINE_RELEASE >= 2018060000
+    if (backendId == DNN_BACKEND_INFERENCE_ENGINE && targetId == DNN_TARGET_MYRIAD &&
+        outChannels == 4 && group == 1 && kernel == Size(3, 1) && pad == Size(0, 1) &&
+        stride == Size(1, 1))
+        throw SkipTestException("Test is disabled for OpenVINO 2018R6");
+#endif
```

Comment:
Can we determine which hyperparameter causes error and skip problem layer is MyriadX is detected?